generated/nifgen/nifgen/session.py

```diff
@@ -3100,7 +3106,9 @@ def __init__(self, resource_name, channel_name=None, reset_device=False, options
         # with the actual session handle.
         self._library_interpreter._vi = self._initialize_with_channels(resource_name, channel_name, reset_device, options)
 
-        self.tclk = nitclk.SessionReference(self._library_interpreter._vi)
+        # NI-TClk does not work over NI gRPC Device Server
```

Comment:
@DavidCurtiss I'm confused by this comment [given that nitclk exists there](https://github.com/ni/grpc-device/tree/main/generated/nitclk). I don't think we need to support fgen-with-tclk-over-grpc in 1.0 but I'm not sure what is expected that wouldn't work.

Reply:
Honestly in looking at the entire file I don't see where the member is even used. Was this some planned usage that didn't get completed?

Reply:
Maybe we should say "isn't yet supported"?

Reply:
How about:
`# NI-TClk isn't yet supported in gRPC-enabled Sessions`
@marcoskirsch ?

Reply:
> I don't see where the member is even used

It's just sitting out there for users to access if they want.

Reply:
sounds good to me

Reply:
> I don't see where the member is even used

This `tclk` member is part of the API.